packages/fxa-settings/public/locales/en-US/settings.ftl

```diff
@@ -247,7 +250,7 @@ delete-account-password-input =
  .label = Enter password
 
 delete-account-cancel-button = Cancel
-delete-account-delete-button = Delete Account
+delete-account-delete-button = Delete
```

Comment:
I think this is a stray line from a different bug

Reply:
I just merged the PR to gitignore this file